RetailSuiteAccounting/UserInterface/01b_Balances.md

```diff
@@ -0,0 +1,44 @@
+# Balances
+
+The *BALANCES* tab offers a T-account view of the current credit and debit situation, and the resulting balance of a selected account. The information displayed in the T-account view is read-only.
+
+
+## Balances view without account selection
+
+*Accounting > Post > Tab BALANCES*
+
+![Balances - no account selected](../../Assets/Screenshots/RetailSuiteAccounting/Book/Balances/BalancesNoAccount.png "[Balances - no account selected]")
+
+When clicking the *BALANCES* tab, a blank tab and the notice "No account selected" may be displayed. An account must be selected to display the corresponding details. If a posting is selected in the *POSTINGS* tab, the account and contra account details corresponding to the selected posting will be displayed in the *BALANCES* tab.
+
+For detailed information about accessing the balance view of an account, see [Access the balance view of an account](../Operation/02_ReviewAccount.md#access-the-balance-view-of-an-account).
+
+## Balances view with account selection
```

Comment:
```suggestion
The *BALANCES* tab offers a T-account view of the current credit and debit situation, and the resulting balance of a selected account.    
An account and/or contra account must be selected to display the corresponding details. Otherwise, the notice *No account selected* is displayed on both sides of the T-view. You can either select an account in the *POST* tab and switch to the *BALANCE* tab or directly enter an account and/or contra account number in the input line in the bottom.

  If a posting is selected in the *POSTINGS* tab, the account and contra account details corresponding to the selected posting will be displayed in the *BALANCES* tab.

For detailed information about accessing the balance view of an account, see [Access the balance view of an account](../Operation/02_ReviewAccount.md#access-the-balance-view-of-an-account).

## Balances view with account selection
```